.github/workflows/plugins-test.0.yaml

```diff
@@ -90,9 +90,10 @@ jobs:
       - uses: actions/checkout@v2
         with:
           submodules: true
-      - uses: actions/setup-java@v1
+      - uses: actions/setup-java@v2
         with:
-          java-version: 8
+          distribution: 'temurin'
```

Comment:
And here, we only use temurin distribution when using Java 17

Reply:
temurin-8 is another distribution of OpenJDK-8, after joining the Eclipse foundation, I think.